Geometry/HcalEventSetup/src/CaloTowerTopologyEP.cc

```diff
@@ -0,0 +1,70 @@
+// -*- C++ -*-
+//
+// Package:    CaloTowerTopologyEP
+// Class:      CaloTowerTopologyEP
+// 
+/**\class CaloTowerTopologyEP CaloTowerTopologyEP.h tmp/CaloTowerTopologyEP/interface/CaloTowerTopologyEP.h
+
+ Description: <one line class summary>
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+// Original Author:  Jeremiah Mans
+//         Created:  Mon Oct  3 11:35:27 CDT 2005
```

Comment:
@ianna - No, but the one it was copied from did... I'll up all the unnecessary comments/variables here.